solr/CHANGES.txt

```diff
@@ -503,13 +503,17 @@ Other Changes
 * SOLR-14858: Add the server WEB-INF/lib directory to the classpath for the solr-exporter script. Will allow the script
   to work when the dist/solrj-lib jars are missing in the Docker image. (Houston Putman)
 
+* SOLR-15124: Removed three core level admin API endpoints because they are already registered at the node level where they
+  really belong: /admin/threads, /admin/properties, /admin/logging (Nazerke Seidan, David Smiley)
+
```

Comment:
@dsmiley should I include this change content in `major-changes-in-solr-9.adoc `?

Reply:
indeed